### PR TITLE
dev-libs/libcgroup: bump 3.1.0, fix CG_CLASSIFY=Yes

### PR DESCRIPTION
**Pull request by negril**
Fixes SRC_URI.

Make CG_CLASSIFY=Yes work by adjusting executable path and add parsing of /etc/cgroup/cgconfig.d.

Bug: https://bugs.gentoo.org/628802
Closes: https://bugs.gentoo.org/628802

### CODE REVIEW

File: dev-libs/libcgroup/files/cgconfig.initd-r1 [hunk 1]
```diff
@@ -1,16 +1,17 @@
 #!/sbin/openrc-run
```
Comment (thesamesam) [inline dev-libs/libcgroup/files/cgconfig.initd-r1:1]: Please copy these (confd+initd) into a new file, or we're going to change the image of 3.0.0 inadvertently

File: dev-libs/libcgroup/libcgroup-3.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		sed -i '/^SUBDIRS/s:tests::' Makefile.am || die
+	fi
+
+	eautoreconf
+}
+
+src_configure() {
+	use elibc_musl && append-ldflags -lfts
+
+	local my_conf=(
```
Comment (thesamesam) [inline dev-libs/libcgroup/libcgroup-3.1.0.ebuild:68]: myeconf, confargs, or myconf if you want, my_conf looks odd

Comment (negril) [reply dev-libs/libcgroup/libcgroup-3.1.0.ebuild]: Is that codified somewhere?

Comment (thesamesam) [reply dev-libs/libcgroup/libcgroup-3.1.0.ebuild]: No, it's just me saying it looks odd ;)

**Pull request closed without merging**